docs/legal/terms.md

```diff
@@ -1,11 +1,29 @@
 # Terms & Conditions
 
-1. No spam/porn/illegal stuff, such as, but not limited to: multi-level marketing schemes. Forums violating these terms will be suspended without prior notice.
-2. Free accounts may include advertisements on its forums.
-3. You are allowed to modify the full appearance of your forum (through the admin panel) but it is not allowed to change appearance as to hide or otherwise hinder the displayed advertisements.
-4. The service applies a fair use policy towards resources (storage, server capacity and traffic) of free accounts. The limits are five times the average of all users of the service. Specifically, this means that no more than 250 users per forum are supported.
-5. Although generous efforts are undertaken to maintain the availability and performance of this service, there is explicitly no warranty given. Use at your own risk.
-6. Forums that are not used for 2 months (no logins/posts/visitors) may be removed.
-7. FreeFlarum.com reserves the right to cancel forums that are detrimental to its quality of service or reputation, at its sole discretion. 
-8. FreeFlarum.com may publicly link to your forum.
-9. This service is not available to persons or legal entities who operate legal constructs for the purpose of tax evasion or concealment of the ultimate benefactor, such to be determined by FreeFlarum.com at its sole discretion.
+!!! info "Consent"
+
+    By continuing to use the FreeFlarum service, you agree with the follwing Terms & Conditions. Forums that violate these regulations will be suspended without prior notice.
+
+**1.** Your forum must not contain spam, porn, or illegal stuff, such as, but not limited to: multi-level marketing schemes, sharing personal details of other people, abuse, hate speech, racism etc... Always check your local laws and regulations before you continue to use this service.
+
+  - FreeFlarum forums are also forbidden to link to external third-party websites & services containing previously mentioned forbidden content.
+
+**2.** FreeFlarum is directed to people who are at least 13 years old or older.
+  - If this server is in the USA, and you are under the age of 13, per the requirements of COPPA (Children's Online Privacy Protection Act), you can't use the FreeFlarum service.
+
+**2.** The service applies a fair use policy towards resources (storage, server capacity and traffic) of free accounts.
+  - The limits are five times the average of all users of the service. Specifically, this means that no more than 250 users per forum are supported.
```

Comment:
How the average of the service's users gets calculated?

Reply:
There is no hard limit, and not any specific calculations. FreeFlarum also runs forums with 250+ users with little no problem, but when you have over 250 users and something starts to break, we can't really help you with that. In average, I'd say that there are at around 40-50 users, so 50×5=250, but I can't really measure the real numbers.